2022-04-17 15:23:28,683 ERROR: Exception on /HOW [POST] [in C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\main\routes.py", line 19, in H2_oil_water
    return jsonify(HOW_pred())
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\data_make.py", line 256, in HOW_pred
    H2LeakageCalSave(hydrogenLeakageVol, collectionTime)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\func_store.py", line 120, in H2LeakageCalSave
    H2LeakageCal + [dict(zip(['time', 'leakage'], [collectionTime] + H2LeakageVol))]
TypeError: can only concatenate list (not "int") to list
2022-04-17 15:23:29,287 ERROR: Exception on /HOW [POST] [in C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\main\routes.py", line 19, in H2_oil_water
    return jsonify(HOW_pred())
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\data_make.py", line 256, in HOW_pred
    H2LeakageCalSave(hydrogenLeakageVol, collectionTime)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\func_store.py", line 120, in H2LeakageCalSave
    H2LeakageCal + [dict(zip(['time', 'leakage'], [collectionTime] + H2LeakageVol))]
TypeError: can only concatenate list (not "int") to list
2022-04-17 15:23:29,839 ERROR: Exception on /HOW [POST] [in C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\main\routes.py", line 19, in H2_oil_water
    return jsonify(HOW_pred())
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\data_make.py", line 256, in HOW_pred
    H2LeakageCalSave(hydrogenLeakageVol, collectionTime)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\func_store.py", line 120, in H2LeakageCalSave
    H2LeakageCal + [dict(zip(['time', 'leakage'], [collectionTime] + H2LeakageVol))]
TypeError: can only concatenate list (not "int") to list
2022-04-17 15:23:50,709 INFO: LaiZhou startup [in C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\__init__.py:34]
2022-04-17 15:25:35,507 INFO: LaiZhou startup [in C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\__init__.py:34]
2022-04-17 15:25:39,963 ERROR: Exception on /HOW [POST] [in C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\main\routes.py", line 19, in H2_oil_water
    return jsonify(HOW_pred())
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\data_make.py", line 256, in HOW_pred
    H2LeakageCalSave(hydrogenLeakageVol, collectionTime)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\func_store.py", line 120, in H2LeakageCalSave
    H2LeakageCal + [dict(zip(['time', 'leakage'], [collectionTime + H2LeakageVol]))]
TypeError: must be str, not int
2022-04-17 15:25:40,533 ERROR: Exception on /HOW [POST] [in C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\main\routes.py", line 19, in H2_oil_water
    return jsonify(HOW_pred())
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\data_make.py", line 256, in HOW_pred
    H2LeakageCalSave(hydrogenLeakageVol, collectionTime)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\func_store.py", line 120, in H2LeakageCalSave
    H2LeakageCal + [dict(zip(['time', 'leakage'], [collectionTime + H2LeakageVol]))]
TypeError: must be str, not int
2022-04-17 15:25:41,118 ERROR: Exception on /HOW [POST] [in C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\main\routes.py", line 19, in H2_oil_water
    return jsonify(HOW_pred())
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\data_make.py", line 256, in HOW_pred
    H2LeakageCalSave(hydrogenLeakageVol, collectionTime)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\func_store.py", line 120, in H2LeakageCalSave
    H2LeakageCal + [dict(zip(['time', 'leakage'], [collectionTime + H2LeakageVol]))]
TypeError: must be str, not int
2022-04-17 15:25:41,685 ERROR: Exception on /HOW [POST] [in C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py:1455]
Traceback (most recent call last):
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 2070, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1515, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1513, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Anaconda3\envs\flask\lib\site-packages\flask\app.py", line 1499, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\main\routes.py", line 19, in H2_oil_water
    return jsonify(HOW_pred())
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\data_make.py", line 256, in HOW_pred
    H2LeakageCalSave(hydrogenLeakageVol, collectionTime)
  File "C:\Users\jsy19\OneDrive - 薅羊毛大学\文档\mystuff\LaiZhouX\app\data\func_store.py", line 120, in H2LeakageCalSave
    H2LeakageCal + [dict(zip(['time', 'leakage'], [collectionTime + H2LeakageVol]))]
TypeError: must be str, not int
